src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const convertedObject = {};
+
+  const splited = sourceString.split(';');
```

Comment:
The variable name 'splited' is a misspelling and does not convey the content of the variable effectively. Consider renaming it to 'declarations' or 'styleDeclarations' to improve readability.